renovate.json5

```diff
@@ -74,6 +74,20 @@
       "groupSlug": "all-non-major-gte-1.0",
       "automerge": true,
     },
+    // We're currently constrained in our ability to update the `tracing`
+    // packages to the latest versions because of our usage.  This should put
+    // those pacakges into their own group and away from other dep updates.
```

Comment:
I wouldn't mind someone offering a more technically clarifying justification than what I've written here (which is a bit hand-wavy, rather than saying *why* we don't update `tracing`.

I also didn't see the justification written down here:

https://github.com/apollographql/router/blob/053c6595f9078c4d3d174c9fce64e5e47d84c735/apollo-router/Cargo.toml#L140-L143

I'm not sure if that reason is already written down somewhere, but this seems like a good place to put it, if not both locations?  Thoughts?

Reply:
yes, that's the right place to insert the messages. There are two problems with tracing updates:
* transient issues due to incompatibilitie between dependencies like https://github.com/apollographql/router/issues/1407 (that we really should dig into soon)
* updating tracing generally means updating tracing-opentelemetry, but we cannot do that until the opentelemetry-* crates that rely on it are updated too, because they depend on a specific version